src/DonationForms/Blocks/DonationFormBlock/Controllers/BlockRenderController.php

```diff
@@ -104,14 +104,6 @@ private function getFormViewUrl(DonationForm $donationForm): string
      */
     protected function loadEmbedScript()
     {
-        (new EnqueueScript(
-            'givewp-donation-form-embed',
-            'build/donationFormEmbed.js',
-            GIVE_PLUGIN_DIR,
-            GIVE_PLUGIN_URL,
-            'give'
-        ))->loadInFooter()->enqueue();
-
```

Comment:
This script is also used in the confirmation page, let's just make sure there are no side effects there (I believe it's loaded seperately)

Reply:
Yes! The videos up top are actually me just testing via different form rendering. At the very end I inspect the Iframe on the confirmation page & you can see the taggedElement and the iframe rendered.